templates/packer/ansible/roles/common/tasks/packages.yml

```diff
@@ -104,11 +105,28 @@
       priority=1
 
 - name: Install worker rpm
+  tags:
+    - rpmcopy
+    - rpmrepo_osbuild
   package:
     name:
       - osbuild-composer-worker
     state: present
 
+- name: Install worker rpm from copr
+  tags:
+    - rpmcopr
+  shell: |
+    dnf copr enable -y @osbuild/osbuild-composer
+    COMPOSER_COMMIT_SHORT=$(echo {{ COMPOSER_COMMIT }} | head -c 9)
+    COMPOSER_RPMS=$(dnf search -q --showduplicates osbuild-composer | grep -E "osbuild-composer-worker-[0-9]+.*g$COMPOSER_COMMIT_SHORT.*{{ ansible_architecture }}" | cut -f 1 -d ':')
+    if [ -z $COMPOSER_RPMS ]; then
+      echo $COMPOSER_COMMIT_SHORT worker rpms not available for on @osbuild/osbuild-composer copr
+      exit 1
+    fi
+    echo "installing rpms $COMPOSER_RPMS"
+    dnf install -y $COMPOSER_RPMS
+
```

Comment:
I guess it's not worth rewriting this more "ansible like" (using the builtin vars and dnf modules)

Reply:
I mean search + grep + cut, it's a hack, idk how you'd rewrite this in the ansible dnf module.

Reply:
Just using the native dnf modules and set-fact (instead of bash-env) as much as possible but probably no real benefit here...